openpype/hosts/traypublisher/plugins/create/create_mov_batch.py

```diff
@@ -0,0 +1,191 @@
+import copy
+import os
+import re
+
+from openpype.client import get_assets
+from openpype.hosts.traypublisher.api import pipeline
+from openpype.lib import FileDef, TextDef, get_subset_name_with_asset_doc
+from openpype.pipeline import (
+    CreatedInstance
+)
+
+from openpype.hosts.traypublisher.api.plugin import TrayPublishCreator
+
+
+class BatchMovCreator(TrayPublishCreator):
+    """Creates instances from .mov file(s)."""
+    identifier = "render_mov_batch"
+    label = "Batch Mov"
+    family = "render"
+    description = "Publish batch of movs"
+    host_name = "traypublisher"
```

Comment:
```suggestion
    host_name = "traypublisher"
```
Already defined from `TrayPublishCreator`